.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -70,7 +70,7 @@ jobs:
   all-in-one:
     name: All in one
     if: inputs.if
-    runs-on: [self-hosted, stackhpc-kayobe-config-aio]
+    runs-on: arc-release-train-runner
```

Comment:
I've installed the github app on this repo, but we'll also need a new runner group for aios defined in the ARC-Installer repo. Then this should reference the name of the new runner group.

Reply:
Ok I will get onto provisioning them.